packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -1289,44 +1818,72 @@ export class ManagedRuleIdentifiers {
+   */
+  public static readonly SSM_DOCUMENT_NOT_PUBLIC = 'SSM_DOCUMENT_NOT_PUBLIC';
+  /**
+   * Checks if a recovery point was created for AWS Storage Gateway volumes.
+   * @see https://docs.aws.amazon.com/config/latest/developerguide/storagegateway-last-backup-recovery-point-created.html
+   */
+  public static readonly STORAGEGATEWAY_LAST_BACKUP_RECOVERY_POINT_CREATED = 'STORAGEGATEWAY_LAST_BACKUP_RECOVERY_POINT_CREATED';
+  /**
+   * hecks if Amazon Virtual Private Cloud (Amazon VPC) subnets are assigned a public IP address.
+   * @see https://docs.aws.amazon.com/config/latest/developerguide/subnet-auto-assign-public-ip-disabled.html
+   */
+  public static readonly SUBNET_AUTO_ASSIGN_PUBLIC_IP_DISABLED = 'SUBNET_AUTO_ASSIGN_PUBLIC_IP_DISABLED';
   /**
    * Checks whether the required public access block settings are configured from account level.
    * @see https://docs.aws.amazon.com/config/latest/developerguide/s3-account-level-public-access-blocks.html
    */
   public static readonly S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS = 'S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS';
   /**
-   * Checks if Amazon Simple Storage Service (Amazon S3) buckets are publicly accessible. This rule is
-   * NON_COMPLIANT if an Amazon S3 bucket is not listed in the excludedPublicBuckets parameter and bucket level
-   * settings are public.
-   * @see https://docs.aws.amazon.com/config/latest/developerguide/s3-bucket-level-public-access-prohibited.html
+   * Checks if the required public access block settings are configured from account level.
+   * @see https://docs.aws.amazon.com/config/latest/developerguide/s3-account-level-public-access-blocks-periodic.html
    */
-  public static readonly S3_BUCKET_LEVEL_PUBLIC_ACCESS_PROHIBITED = 'S3_BUCKET_LEVEL_PUBLIC_ACCESS_PROHIBITED';
+  public static readonly S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS_PERIODIC = 'S3_ACCOUNT_LEVEL_PUBLIC_ACCESS_BLOCKS_PERIODIC';
   /**
-   * Checks that the Amazon Simple Storage Service bucket policy does not allow
-   * blocked bucket-level and object-level actions on resources in the bucket
-   * for principals from other AWS accounts.
+   * Checks if Amazon Simple Storage Service (Amazon S3) Buckets allow user permissions through access control lists (ACLs).
+   * @see https://docs.aws.amazon.com/config/latest/developerguide/s3-bucket-acl-prohibited.html
+   */
+  public static readonly S3_BUCKET_ACL_PROHIBITED = 'S3_BUCKET_ACL_PROHIBITED';
+  /**
+   * Checks if the Amazon Simple Storage Service bucket policy does not allow blacklisted bucket-level
+   * and object-level actions on resources in the bucket for principals from other AWS accounts.
    * @see https://docs.aws.amazon.com/config/latest/developerguide/s3-bucket-blacklisted-actions-prohibited.html
    */
-  public static readonly S3_BUCKET_BLOCKED_ACTIONS_PROHIBITED = 'S3_BUCKET_BLACKLISTED_ACTIONS_PROHIBITED';
+  public static readonly S3_BUCKET_BLACKLISTED_ACTIONS_PROHIBITED = 'S3_BUCKET_BLACKLISTED_ACTIONS_PROHIBITED';
```

Comment:
As a policy, we now use `BLOCKED`. This one shouldn't change.

Reply:
Also, I think that renaming this would cause breaking changes for anyone using it.

Reply:
Undo any changes that are expected to be destructive. Should we add element names in addition to that that conform to the correct identifier naming?

Reply:
In this case, the different term was used because we don't want to use a non-inclusive term. `blocked` is the correct replacement here.